permissions/plugin-configuration-as-code.yml

```
@@ -11,7 +11,9 @@ developers:
 - "timja"
 security:
   contacts:
-    jira: "jetersen"
+    jira: 
+       - "jetersen"
+       - "foundation_security_members"
```

your format wasn't valid
```suggestion
    jira: "foundation_security_members"
```

---

Thanks for the heads up

---

Is it possible to have both?

---

no you can't have both without re-working the code, it only handles one.